_data/sahirmpatel/sahirmpatel.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sahir Patel
+institution: RAIT
+profile_pic: sahirmpatel.png 
```

Comment:
Your profile picture path does not match the current file name (```sahirmpatel.jpg```).

Reply:
Oops , changed it now